lib/formatItem.js

```diff
@@ -18,9 +18,8 @@ module.exports = (DocumentClient) => (attributes,linked,item,include=[]) => {
   const validateType = validateTypes(DocumentClient)
 
   return Object.keys(item).reduce((acc,field) => {
-
-    if (linked[field] || linked[attributes[field].alias]) {
-      Object.assign(acc, (linked[field] || linked[attributes[field].alias]).reduce((acc,f,i) => {
+    if (linked[field]) {
+      Object.assign(acc, (linked[field]).reduce((acc,f,i) => {
```

Comment:
I was trying to figure out a test case would cover the linked alias branch that was erroring out and I _think_ the only way it can be reached is by using an alias when referring to composite key, which is not allowed.
So I ended up taking out that code path.